test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,15 +1,43 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-
 import java.util.Arrays;
 import java.util.List;
 
 import org.junit.Test;
 
+import static org.junit.Assert.*;
+
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        //Solution below adapted from the current file, UtilsTest.java (line 44-64) and from
+        //https://stackoverflow.com/questions/7123555/assertnull-should-be-used-or-assertnotnull
+
+        // empty
+        assertNotNull();
+        assertNotNull("");
+
+        // check null or mix of null with object
+        assertNull(null,null);
+        assertNull("123",null);
+        assertNull((Object) null);
+        assertNull("test",null,null);
+
+        // check not null
+        assertNotNull(2);
+        assertNotNull("abc","123");
+        assertNotNull("def");
+
+    }
+
+    private void assertNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNotNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```

Comment:
It's good to follow the style used by the code nearby 👍
Having said that, it is doubtful if extracting these two methods add any value.
In assertAreUnique and assertNotUnique refactoring helps to convert arrays passed to the method as collections.
For isAnyNull refactoring just adds on to the height of the code.